dev/com.ibm.ws.messaging.jms.2.0.cdi/bnd.bnd

```diff
@@ -34,6 +34,6 @@ instrument.classesExcludes: com/ibm/ws/cdi/jms/resources/*.class
 	com.ibm.websphere.appserver.spi.logging;version=latest,\
 	com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
 	com.ibm.ws.cdi.interfaces;version=latest,\
-	com.ibm.websphere.javaee.cdi.1.2;version=latest,\
+	com.ibm.websphere.javaee.cdi.2.0;version=latest,\
```

Comment:
Shouldn't this remain as cdi 1.2?

Reply:
and then the Import-Package statement should be expanded to include cdi-2.0 package versions

Reply:
yes, I'll get that and the import version ranges updated